packages/plugin-ext-vscode/src/node/scanner-vscode.ts

```diff
@@ -28,7 +28,7 @@ export class VsCodePluginScanner extends TheiaPluginScanner implements PluginSca
 
     getModel(plugin: PluginPackage): PluginModel {
         const result: PluginModel = {
-            id: `${plugin.publisher}.${plugin.name}`,
+            id: `${plugin.publisher.toLowerCase()}.${plugin.name.toLowerCase()}`,
```

Comment:
why toLowerCase ?

Reply:
https://github.com/microsoft/vscode/blob/15916055fe0cb9411a5f36119b3b012458fe0a1d/src/vs/platform/extensions/common/extensions.ts#L167-L169 because extension can be referenced only in the lower case, but published information is not mandatory in the lower case

Reply:
probable better to put as the comment in the code

Reply:
I've added comments to ID definition in VS Code repo.